ow_dynamic_terrain/scripts/modify_terrain_scoop_pub.py

```diff
@@ -15,23 +15,24 @@
 from tf.transformations import euler_from_quaternion
 from ow_dynamic_terrain.msg import modify_terrain_ellipse
 
-
 class ModifyTerrainScoop:
 
   def __init__(self, *args):
     rospy.init_node("modify_terrain_scoop_pub", anonymous=True)
     self.last_translation = np.zeros(3)
-    self.pub = rospy.Publisher(
-        'ow_dynamic_terrain/modify_terrain_ellipse', modify_terrain_ellipse, queue_size=1)
+    self.visual_pub = rospy.Publisher(
+        'ow_dynamic_terrain/modify_terrain_ellipse/visual', modify_terrain_ellipse, queue_size=1)
+    self.collision_pub = rospy.Publisher(
+        'ow_dynamic_terrain/modify_terrain_ellipse/collision', modify_terrain_ellipse, queue_size=1)
     self.states_sub = rospy.Subscriber(
         "/gazebo/link_states", LinkStates, self.handle_link_states)
 
-  def compose_modify_terrain_ellipse_message(self, position, orientation):
+  def compose_modify_terrain_ellipse_message(self, position, orientation, scale=1.0):
     return modify_terrain_ellipse(position=Point(position.x, position.y, position.z),
                                   orientation=orientation,
-                                  outer_radius_a=0.002, outer_radius_b=0.005,
-                                  inner_radius_a=0.001, inner_radius_b=0.001,
-                                  weight=-0.025, merge_method="min")
+                                  outer_radius_a=0.002*scale, outer_radius_b=0.005*scale,
```

Comment:
Where do the numbers 0.002, 0.005, and 0.001 come from? Please add a comment to explain the logic here.

Reply:
addressed